doc/glossary.rst

```diff
@@ -122,6 +122,13 @@ reflect either technical terms or legacy ways of referring to Ceph systems.
 		The Ceph manager software, which collects all the state from the whole
 		cluster in one place.
 
+	Ceph Manager Dashboard
+	Ceph Dashboard
+	Dashboard Plugin
+	Dashboard
+                A native web based monitoring and administration application to
```

Comment:
I'm not sure "native" makes sense in this context. How about "built-in"? Also, would it make sense to add a pointer to the documentation page here?